source/connect/client.txt

```diff
@@ -0,0 +1,110 @@
+.. _php-client:
+
+=======================
+Create a MongoDB Client
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoDB\\Client** object, which creates the connection to the MongoDB deployment
+  and lets you perform operations on it.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoDB\Client`` object
+to connect to MongoDB.
+
+.. _php-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
```

Comment:
S: change to ``db_username`` and ``db_password``

Reply:
do these need to be disambiguated from other credentials? the following paragraph also says they're for the DB.

Reply:
I think it could be either. Just figured we might start integrating those placeholders where applicable so that there are fewer mis-matches

Reply:
Have you considered including that connection string image on this page and using those placeholders instead? another option

Reply:
Not opposed to this change, but should we be consistent with the Server manual?

https://www.mongodb.com/docs/manual/reference/connection-string/#connection-string-components

Reply:
I'm a bit late here, but I don't think these should be `db_username` and `db_password`. DB-backed auth is but one mechanism. These could be credentials for a different auth mechanism, so more generic names are appropriate.

See also: [connection string spec](https://github.com/mongodb/specifications/blob/master/source/connection-string/connection-string-spec.md)